mpas_analysis/ocean/conservation.py

```diff
@@ -0,0 +1,626 @@
+            The type of plot to generate from conservationCheck variables
+        """
+        config = self.config
+        filePrefix = self.filePrefixes[plot_type]
+        outFileName = f'{self.plotsDirectory}/{filePrefix}.png'
+
+        titles = {}
+        titles['total_energy_flux'] = 'Total energy flux'
+        titles['absolute_energy_error'] = 'Energy error'
+        titles['ice_salt_flux'] = 'Salt flux related to land ice and sea ice'
+        titles['absolute_salt_error'] = 'Salt conservation error'
+        titles['total_mass_flux'] = 'Total mass flux'
+        titles['total_mass_change'] = 'Total mass anomaly'
+        titles['land_ice_mass_flux'] = 'Mass flux due to land ice'
+        titles['land_ice_mass_change'] = 'Mass anomaly due to land ice fluxes'
+        titles['land_ice_ssh_change'] = 'SSH anomaly due to land ice fluxes'
+        titles['land_ice_mass_flux_components'] = 'Mass fluxes from land ice'
+
+        y_labels = {}
+        y_labels['total_energy_flux'] = 'Energy flux (W)'
+        y_labels['absolute_energy_error'] = 'Energy (J)'
+        y_labels['ice_salt_flux'] = 'Salt flux (Gt/yr)'
+        y_labels['absolute_salt_error'] = 'Salt (Gt)'
+        y_labels['total_mass_flux'] = 'Mass flux (Gt/yr)'
+        y_labels['total_mass_change'] = 'Mass (Gt)'
+        y_labels['land_ice_mass_flux'] = 'Mass flux (Gt/yr)'
+        y_labels['land_ice_mass_change'] = 'Mass (Gt)'
+        y_labels['land_ice_ssh_change'] = 'SSH anomaly (mm)'
+        y_labels['land_ice_mass_flux_components'] = 'Mass flux (Gt/yr)'
+
+        captions = {}
+        captions['total_energy_flux'] = 'Total energy flux'
+        captions['absolute_energy_error'] = 'Absolute energy conservation error'
+        captions['ice_salt_flux'] = 'Salt flux related to land ice and sea ice ' \
+            '(sea ice salinity flux, sea ice frazil flux, and land ice frazil flux)'
+        captions['absolute_salt_error'] = 'Absolute salt conservation error'
+        captions['total_mass_flux'] = 'Total mass flux'
+        captions['total_mass_change'] = 'Total mass anomaly'
+        captions['land_ice_mass_flux'] = 'Mass flux due to land ice'
+        captions['land_ice_mass_change'] = 'Mass anomaly due to land ice fluxes'
+        captions['land_ice_ssh_change'] = 'SSH anomaly due to land ice fluxes. Assumes a constant ocean area.'
+        captions['land_ice_mass_flux_components'] = 'Mass flux components from land ice'
+
+        self.logger.info(f'  Open conservation file {self.outputFile}...')
+        ds = open_mpas_dataset(fileName=self.outputFile,
+                               calendar=self.calendar,
+                               variableList=self.variableList[plot_type],
+                               timeVariableNames='xtime',
+                               startDate=f'{self.startYear:04d}-01-01_00:00:00',
+                               endDate=f'{self.endYear:04d}-01-01_00:00:00')
+
+        if self.controlConfig is not 'None':
```

Comment:
None should not be in quotes here.

Reply:
Thanks for catching that! I'll test with and without control before merging.